examples/grpc-server/message.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";

```

Comment:
I would comment this file and it should conform to buf linting standards: https://buf.build/docs/lint/usage/

(also this file should be formatted).

Reply:
What is buf linting use for? Some rule of it seems to be weird.
like
"llama.v1.Request" is used as the request or response type for multiple RPCs.

why reusing same type is not allowed?

Reply:
@extrame - Here is a [list of rules](https://buf.build/docs/lint/rules/), and "why" answers.

When I first found `buf` (about 4-5 years ago?), I was initially looking at how to disable many of the lint rules (which I do not recommend), but after reviewing that document, I became convinced that the rules were all there for **really really good reasons** that I couldn't possibly have foreseen developing for my initial use case... In the end, as my services evolved to serve a diverse set of clients in multiple languages, having adopted `buf` early on, made my life much easier. Given that this is a popular open source project an unlimited number of potential future and diverse clients, this may become the case here rapidly.

Ever since then, I've been using it regularly, and continue to do so.

The RPC API will change, and it's best to be prepared for it from the start.

Reply:
Echoing @canardleteer a bit here but the buf rules are born out of many years of practical proto use both from google and uber. I don't mean to appeal to authority here but please look into them as many are rooted in wisdom gleaned from practical application over time.